packages/flutter/lib/src/gestures/converter.dart

```diff
@@ -32,6 +32,25 @@ class _PointerState {
 
   Offset lastPosition;
 
+  Offset deltaTo(Offset to) {
+    if (to == null) { // [PointerRemovedEvent]
```

Comment:
i don't understand what this comment means

Reply:
The only case where `PointerEvent.position` would be null is `PointerRemovedEvent`.